salt/modules/cmdmod.py

```diff
@@ -372,10 +375,11 @@ def _get_stripped(cmd):
         # requested. The command output is what will be controlled by the
         # 'loglevel' parameter.
         msg = (
-            'Executing command {0}{1}{0} {2}in directory \'{3}\'{4}'.format(
+            u'Executing command {0}{1}{0} {2}{3}in directory \'{4}\'{5}'.format(
```

Comment:
The `u` is unnecessary here since we're going to be using `unicode_literals`.

Reply:
I should have noticed it wasn't in the original string (or rather, not the one there after rebasing). Sorry I didn't pick up on that. Fixed it anyway.